oaknational/Oak-Web-Application
# [pupil-801] Adding ids to OakInfo components to resolve pa11y issues

## Description

Music year: 1963

Upgrading Oak components with fix for pa11y issue on pupil pages.

## How to test

1. Goto - these urls and there should be no pa11y issues. 

- https://deploy-preview-2533--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-primary-year-5/units/multiplication-by-partitioning-leading-to-short-multiplication-3-by-1-digit/lessons 
- https://deploy-preview-2533--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-primary-year-5/units
- https://deploy-preview-2533--oak-web-application.netlify.thenational.academy/pupils/programmes/combined-science-secondary-year-10/options

## Checklist
